src/client/ContentManagement.js

```diff
@@ -1600,3 +1600,69 @@ exports.DeleteWriteToken = async function({writeToken, libraryId}) {
 
   await this.HttpClient.ClearWriteToken({writeToken});
 };
+
+
+
+
+/**
+ * Get node information for a write token
+ *
+ * @methodGroup Content Objects
+ * @namedParams
+ * @param {string} writeToken - write token to look up
+ *
+ * @returns {Promise<object>} - Response containing info for the write token's node
+ */
+exports.WriteTokenNodeInfo = async function({writeToken}) {
+  ValidateWriteToken(writeToken);
+
+  this.Log(`Looking up node for write token: ${writeToken}`);
+
+  // For now, no auth required for this call
+
+  // const authToken = await this.authClient.AuthorizationToken({
+  //   noAuth: true  // Don't create an accessRequest blockchain transaction
+  // });
+  
+  const response = await this.utils.ResponseToJson(
+    this.HttpClient.Request({
+      path: "nodes",
+      method: "GET",
+      headers: {
+        Authorization: `Bearer ${authToken}`
+      }
+      // queryParams: {
+      //   token: writeToken
+      // }
+    })
+  );
+
+  if(response.nodes.length === 0) {
+    throw Error("No node found for write token");
+  }
+
+  if(response.nodes.length > 1) {
+    throw Error(`Unexpected number of nodes found (${list.length})`);
+  }
+
+  return response.nodes[0];
+};
+
+/**
+ * Get node URL for a write token
+ *
```

Comment:
I am on the fence on this - the API already redirects to the correct node which makes the API and the client SDK more ergonomic.  It's not bad to have a function to retrieve the node and URL endpoint for a write token though.

I also think we have another client API to retrieve the ingest node for a live stream edge write token - would be good to unify in case there are two ways of doing it (cc @elv-zenia )